app/crud/crud_test_run_execution.py

```diff
@@ -182,13 +181,9 @@ def create(
         # TODO: while we change the API. selected tests can come from two places:
         # 1. Pass in directly
         # 2. from the optional test_run_config
-        selected_tests: Optional[TestSelection] = kwargs.get("selected_tests")
+        selected_tests: Optional[SelectedTests] = kwargs.get("selected_tests")
```

Comment:
Should this be optional? Or should there be an error if no test is selected?
Do you know why this method uses `kwargs` instead of using `selected_tests` directly?

Reply:
Using kwargs, was a way to support both 
 1. Pass in directly
 2. from the optional test_run_config
 
I hope we can soon clear this up, and put selected tests directly on TestRunExecutionCreate.

Reply:
As soon as the PR https://github.com/project-chip/certification-tool-backend/pull/6 enters, selected tests will be mandatory and I'll create another PR that hopefully will remove completely the `TestRunConfig` and this kwarg.
Unfortunately, it is taking some time for the Frontend counterpart to be merged (https://github.com/project-chip/certification-tool-frontend/pull/6)

Reply:
The above optional section was removed. I'll close this but let me know in the updated code